cli/schemas/environment/environment.go

```diff
@@ -8,6 +8,7 @@ import (
 //Environment represents a environment.yml file
 type Environment struct {
 	Name     string    `yaml:"name,omitempty"`
+	Root     *Provider `yaml:"root,omitempty"`
```

Comment:
I was thinking if we should create a DNSProvider struct so it's more clear which info comes here

Reply:
Yeap, let's go with that